lib/LWP/UserAgent.pm

```diff
@@ -1133,22 +1133,48 @@ LWP::UserAgent - Web user agent class
 
 =head1 SYNOPSIS
 
- use strict;
- use warnings;
- use LWP::UserAgent ();
+    use strict;
+    use warnings;
 
- my $ua = LWP::UserAgent->new;
- $ua->timeout(10);
- $ua->env_proxy;
+    use LWP::UserAgent ();
 
- my $response = $ua->get('http://example.com');
+    my $ua = LWP::UserAgent->new(timeout => 10);
+    $ua->env_proxy;
+
+    my $response = $ua->get('http://example.com');
+
+    if ($response->is_success) {
+        print $response->decoded_content;    # or whatever
+    }
+    else {
+        die $response->status_line;
+    }
+
+Extra layers of security (note the C<cookie_jar> and C<protocols_allowed>):
+
+    use strict;
+    use warnings;
+
+    use HTTP::CookieJar::LWP ();
+    use LWP::UserAgent       ();
 
- if ($response->is_success) {
-     print $response->decoded_content;  # or whatever
- }
- else {
-     die $response->status_line;
- }
+    my $jar   = HTTP::CookieJar::LWP->new;
+    my $agent = LWP::UserAgent->new(
+        cookie_jar        => $jar,
+        protocols_allowed => ['http', 'https'],
+        timeout           => 10,
+    );
+
+    $ua->env_proxy;
+
+    my $response = $ua->get('http://example.com');
+
+    if ($response->is_success) {
+        print $response->decoded_content;    # or whatever
```

Comment:
It can do without the `# or whatever` comment

I usually use the `...` to indicate arbitrary code that can be anything before or after ... or here, in the middle

 But having the `decoded_content` is a good example too

Reply:
I agree, I'm not sure why someone *cough* *cough* ever put that in there. It reads oddly passive aggressive. /me hides